docs/reference/components/sources/file.cue

```diff
@@ -121,6 +121,12 @@ components: sources: file: {
 			required:    false
 			type: string: default: "host"
 		}
+		ignore_not_found: {
+			common:      false
+			description: "Ignore missing files when fingerprinting. This can be used to reduce errors logged when reading from files that may include dangling symlinks."
```

Comment:
```suggestion
			description: "Ignore missing files when fingerprinting. This may be useful when used with source directories containing dangling symlinks."
```